docs/start.rst

```diff
@@ -13,7 +13,7 @@ The installation is quick and straightforward.
     # install from pypi using pip
     pip install airflow
 
-    # initialize the database
```

Comment:
This will initialize any database that you have specified in airflow.cfg 

For example, by default, it will be set up to point to SQLite
sql_alchemy_conn = sqlite:////Users/siddharth/Projects/airflow/airflow.db

However, I can also change it access my local Postgres instance.
sql_alchemy_conn = postgresql://siddharth@localhost:5432/airflow

Hence, the "sqlite" term here is incorrect.